db/seeds.rb

```diff
@@ -55,11 +55,13 @@
   fixture_ids_table = CSV.read(oid_path, headers: true)
   oids = fixture_ids_table.by_col[0]
 
-  oids.each do |row|
-    po = ParentObject.new
-    po.oid = row
-    po.save
-    puts "#{po.oid} saved"
+  unless ParentObject.count > 0
```

Comment:
My understanding for this ticket was that this was supposed to follow the pattern for the other seeds. 

If we did that, a developer could add an example oid or two to the CSV, re-run the seed file, and have those in their local environment, which I could see as being really helpful for fast development.